numba/_helperlib.c

```diff
@@ -8,17 +8,21 @@
 #include <stddef.h>
 #include <stdio.h>
 #include <math.h>
+#include <complex.h>
 #ifdef _MSC_VER
     #define int64_t signed __int64
     #define uint64_t unsigned __int64
     #define uint32_t unsigned __int32
+    #define _complex_float _Fcomplex
```

Comment:
```suggestion
    #define _complex_float_t _Fcomplex
```
maybe name with `_t` suffix to indicate type?